app/models/City.rb

```diff
@@ -1,4 +1,5 @@
 class City < ActiveRecord::Base
+  searchkick autocomplete: ['city']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.